src/AWSLambda/Events/SNSEvent.hs

```diff
@@ -37,20 +37,20 @@ $(makeLenses ''MessageAttribute)
 
 data SNSMessage message = SNSMessage
   { _smMessage           :: !(TextValue message )
-  , _smMessageAttributes :: !(HashMap Text MessageAttribute)
+  , _smMessageAttributes :: !(Maybe (HashMap Text MessageAttribute))
   , _smMessageId         :: !Text
   , _smSignature         :: !Text
   , _smSignatureVersion  :: !Text
-  , _smSigningCertUrl    :: !Text
+  , _smSigningCertURL    :: !Text
   , _smSubject           :: !Text
   , _smTimestamp         :: !UTCTime
   , _smTopicArn          :: !Text
```

Comment:
This is strange, because it (i.e. the feeder lambda) was working with `_smSigningCertUrl` and `_smUnsubscribeUrl` before Tom changed it to use SQS. I haven't investigated, but is it possible that the fields are different depending on whether the SNS event is wrapped in an SQS event or not? If so, a possible solution would be to make the field parsing case-insensitive.